app/(main)/league/[leagueId]/entry/all/page.tsx

```diff
@@ -87,7 +91,6 @@ const Entry = ({
     if (!user.id || user.id === '') {
```

Comment:
It seems like this if statement is an exact copy of what you added. This seems redundant and we should probably be deciding which one to keep

Reply:
Removed this one too